README.md

```diff
@@ -168,6 +168,7 @@ From the KRACK <a href="https://www.krackattacks.com/">website</a>:
 | SUSE / openSUSE | [LINK](https://bugzilla.suse.com/show_bug.cgi?id=1063479) |  | 2017-10-16 | 2017-10-16 | 28 Aug 2017 |
 | Swisscom | [LINK](https://supportcommunity.swisscom.ch/t5/Diskussionen-%C3%BCber-Ger%C3%A4te-und/WPA2-Leack-wirklich-oder-wieder-nur-Baitfishing/m-p/511992#M18540) | Internet Box routers not affected. Centro routers and AirTies repeaters to be clarified. | 2017-10-17 | 2017-10-17 | |
 | Synology | [LINK](https://www.synology.com/en-us/support/security/Synology_SA_17_60_KRACK) | Synology DiskStation Manager (DSM) with attached WiFi dongle and Synology Router Manager (SRM) are vulnerable to Krack. According to Synology, updates for affected products will be released soon. | 2017-10-17 | 2017-10-17 |  |
+| Tesco | [LINK](https://twitter.com/Tesco/status/920254455135854592) | "There will be no further updates to the hudl software" | 2017-10-17 | 2017-10-17 |  |
```

Comment:
Can we add to comments a more clear message: "tesco has chosen not to patch the Hudl" ?

Reply:
@kristate Why not? It helps.